lib/stub_ui/lib/painting.dart

```diff
@@ -3312,6 +3320,15 @@ class Picture {
   Future<Image> toImage(int width, int height) {
     throw UnimplementedError();
   }
+
+  /// Creates a picture shader from this picture.
+  ///
+  /// The shader is returned asynchronously to allow time for the gpu to
+  /// draw the picture and compile a shader.
+  Future<PictureShader> toShader(TileMode tmx, TileMode tmy, Float64List matrix4) {
```

Comment:
In general we prefer to avoid abbreviations, so `tileModeX` or some such.

In https://master-api.flutter.dev/flutter/painting/paintImage.html we use `RepeatImage` which doesn't expose `mirror`. Is your goal to eventually add `mirror` values to the `RepeatImage` enum?